src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -72,7 +72,16 @@ public MakelistCmd()
         protected void execute(CommandEvent event) 
         {
             String pname = event.getArgs().replaceAll("\\s+", "_");
-            if(bot.getPlaylistLoader().getPlaylist(pname)==null)
+            if(pname == null || pname.isEmpty()) 
+            {
+                event.reply(event.getClient().getError() + " Please provide a name for playlist!");
```

Comment:
Old code may still use this, but you should use `event.replyError` instead. Additionally, the grammar should be slightly adjusted:
```suggestion
                event.replyError("Please provide a name for the playlist!");
```